src/main/lrsql/ops/query/generate.clj

```diff
@@ -0,0 +1,16 @@
+(ns lrsql.ops.query.generate
```

Comment:
The ops/(query|command) namespaces are generally for things that do queries, not pure data stuff like this. I'd suggest moving this somewhere like `lrsql.util.reaction`

Reply:
moved